src/handbook/company/security/incident-response.md

```diff
@@ -0,0 +1,163 @@
+---
+navTitle: Incident Response Plan
+---
+
+# Data Management Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-06-01     |
+
+## Purpose
+
+This document establishes the plan for managing information security incidents
+and events, and offers guidance for employees or incident responders who believe
+they have discovered, or are responding to, a security incident.
+
+## Scope 
+
+This policy covers all information security or data privacy events or incidents.
+
+## Incident and Event Definitions
+
+A **security event** is an observable occurrence relevant to the confidentiality,
+availability, integrity, or privacy of company controlled data, systems or networks.
+
+A **security incident** is a *security event* which results in loss or damage to
+the confidentiality, availability, integrity, or privacy of company controlled
+data, systems or networks.
+
+## Security Incident Response Team (SIRT)
+
+The Security Incident Response Team (SIRT) is responsible for
+
+ - Reviewing analyzing, and logging all received reports and tracking their statuses.
+ - Performing investigations, creating and executing action plans, and post-incident activities.
+ - Collaboration with law enforcement agencies.
+
+Current members of the FlowForge SIRT:
+
+ - CTO
+ - CEO
+ - Customer Success manager
+ - Product Owner
+
+
+## Incident Reporting & Documentation
```

Comment:
@knolleary We have a policy, please merge these

Reply:
Just to clarify, are you referring to https://flowforge.com/handbook/development/security ?

I saw that more as our outward facing policy for 3rd parties to report issues - rather than our internal security incident policy. But I agree there should be some clarify and cross-referencing.

Reply:
Have linked the policies together - keeping one focussed on how external parties can report issues (inc bug bounty) and the other on how we handle things internally. I think that is the right split to have.